app/(app)/tickets/shorten/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query'
+import { router, useLocalSearchParams } from 'expo-router'
+import { SafeAreaView, ScrollView, View } from 'react-native'
+
+import ScreenContent from '@/components/screen-layout/ScreenContent'
+import ScreenView from '@/components/screen-layout/ScreenView'
+import Button from '@/components/shared/Button'
+import Markdown from '@/components/shared/Markdown'
+import Typography from '@/components/shared/Typography'
+import { useTranslation } from '@/hooks/useTranslation'
+import { clientApi } from '@/modules/backend/client-api'
+
+const Shorten = () => {
+  const t = useTranslation('ShortenTicket')
+  const queryClient = useQueryClient()
+
+  const { ticketId } = useLocalSearchParams<{ ticketId: string }>()
+
+  if (!ticketId) {
+    router.replace('/tickets')
+  }
+
+  const shortenTicketMutation = useMutation({
+    mutationFn: (id: number) => clientApi.ticketsControllerShortenTicket(id),
+    onSuccess: () => {
+      queryClient.removeQueries({ queryKey: ['Tickets'] })
```

Comment:
The `queryKey` should be taken from the `activeTicketsOptions`, so that we do not lose track of where the keys are used.

Reply:
This is different, because I want to remove all the queries that start with `"Tickets"` key, not just one query